crates/fluvio-auth/src/x509/authenticator.rs

```diff
@@ -36,15 +36,15 @@ impl ScopeBindings {
 
 #[derive(Debug)]
 pub struct X509Authenticator {
```

Comment:
Initial scope of the authenticator was for only for SC.   Should not overload these with two different purpose. Probably simpler rename original `X509Authenticator` as `X509AuthenticatorSC` and move to `SC`.   And use this crate only for common auth utilities and libraries.

Reply:
based on other conversation, this should need to be changed since `remote` can be added to scope